README.md

```diff
@@ -134,6 +134,21 @@ Analyzer is installed. For example, on Windows the default installation
 location is C:\\Program
 Files\\Fortify\\Fortify\_SCA\_and\_Apps\_*\<version\>*.
 
+### Configuration when running Jenkins from the Docker
```

Comment:
Question:
Is the following description generated text or you made? If you made, I consulted our doc owner Karene, and she provided the following.

Reply:
I made description by myself. Of course, opinions are welcome 😄